invenio_users_resources/services/schemas.py

```diff
@@ -100,7 +101,7 @@ class UserSchema(BaseRecordSchema, FieldPermissionsMixin):
     visibility = fields.Str(dump_only=True)
     is_current_user = fields.Method("is_self", dump_only=True)
 
-    email = fields.String()
+    email = fields.Email(required=True)
```

Comment:
is this a change which is backwards incompatible?

Reply:
That's a good point. It seems that both OAuth and SAML rely on the `security` [registration form](https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/utils.py#L137), which has a similar [email validation](https://github.com/inveniosoftware/flask-security-fork/blob/master/flask_security/forms.py#L124) as the create account one.

I think _it should be ok_ (famous last words)